test-ca/build-a-pki.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
I made this change because my IDE was warning me that apparently sh + POSIX doesn't standardize the -n flag for echo. I need echo -n because writing a \n to index.txt causes openssl ca to barf parsing the db state.

In general it seemed fine to assume bash here.